client/packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -26,7 +26,7 @@ import { useFormatDateTime, useTranslation } from '@common/intl';
 import { useTableStore } from './context';
 
 interface RenderRowsProps<T extends RecordWithId> {
-  ref: React.RefObject<HTMLDivElement>;
+  mRef: React.RefObject<HTMLDivElement>;
   data: T[];
```

Comment:
This is to fix console error for datatable component.
<img width="759" alt="Screenshot 2023-10-07 at 10 32 45 AM" src="https://github.com/openmsupply/open-msupply/assets/13145522/1753209e-372a-4f04-b472-c61a437a5a99">

Reply:
good spotting! thanks